### PR TITLE
Add admin

### PR DESCRIPTION
**Pull request by NeverSayNeverLove**

### CODE REVIEW

File: app/views/users/_user.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <li>
-  <%= link_to user.name, user %>
+  <%= link_to user.email, user %>
   <% if current_user.admin? && !current_user?(user) %>
```
Comment (peiguangda) [inline app/views/users/_user.html.erb]: bo ngoac

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+    <table class="table table-hover">
+      <thead>
+        <tr>
+          <th><%= link_to t(".email") %></th>
+          <th><%= link_to t(".role") %></th>
+          <th></th>
+        </tr>
+      </thead>
+      <% @users.each do |user| %>
+        <% if !current_user?(user) %>
```
Comment (peiguangda) [inline app/views/users/index.html.erb]: bo ngoac, dung unless

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
+  activated: true,
+  activated_at: Time.zone.now)
+# User.create!(email: "example@railstutorial.org",
+#   password: "foobar",
+#   password_confirmation: "foobar",
+#   admin: true)
+# User.create!(email: "example@railstutorial.org",
+#   password: "foobar",
+#   password_confirmation: "foobar",
+#   admin: true)
```
Comment (peiguangda) [inline db/seeds.rb]: xuong dong, xoa cmt

**Comment by hoangvietthai** [comment]
ok

**Comment by peiguangda** [comment]
ok baby

**Pull request merged**